addons/mass_mailing_sms/models/mailing_mailing.py

```diff
@@ -82,6 +84,14 @@ def _compute_sms_has_iap_failure(self):
                 mail.sms_has_insufficient_credit = trace_dict[mail.id]['sms_credit']
                 mail.sms_has_unregistered_account = trace_dict[mail.id]['sms_acc']
 
+    @api.constrains('ab_testing_pc')
+    def _check_total_percentage_ab_test(self):
```

Comment:
Is it intended to separate mail and sms on the same campaign computation ? Quite strange computation...